integration/nestjs-simple/hero.ts

```diff
@@ -49,13 +49,13 @@ export interface HeroServiceClient {
 export interface HeroServiceController {
   addOneHero(request: Hero): void;
 
-  findOneHero(request: HeroById): Promise<Hero> | Observable<Hero> | Hero;
+  findOneHero(request: HeroById): Promise<Hero> | Hero;
 
-  findOneVillain(request: VillainById): Promise<Villain> | Observable<Villain> | Villain;
+  findOneVillain(request: VillainById): Promise<Villain> | Villain;
 
   findManyVillain(request: Observable<VillainById>): Observable<Villain>;
 
-  findManyVillainStreamIn(request: Observable<VillainById>): Promise<Villain> | Observable<Villain> | Villain;
+  findManyVillainStreamIn(request: Observable<VillainById>): Promise<Villain> | Villain;
```

Comment:
@alexkb Disclaimer that I don't use NestJS, but wouldn't just `Promise<Villain>` make more sense here as a return type?

I agree that your PR is already better than the previous `Observable<T> | Promise<T> | T`, which imo seems just kind of maddening to use as a caller, b/c you'd have to constantly be checking what the return value was.